scripts/bash/gitlab_migrate.sh

```diff
@@ -164,14 +164,16 @@ sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'CI_EXTERNAL_URL', value
 sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'CI_SERVER_HOST', value: '$INSTANCE_INTERNAL_HOSTNAME').save"
 sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'CI_SERVER_URL', value: '$INSTANCE_INTERNAL_URL').save"
 sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'CI_API_V4_URL', value: '$INSTANCE_INTERNAL_API_V4_URL').save"
-sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'GCP_PROJECT_ID', value: '$GCP_PROJECT_ID').save"
+sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'CONTAINER_REGISTRY_NAMESPACE ', value: '$GCP_PROJECT_ID').save"
+sudo gitlab-rails runner "Ci::InstanceVariable.new(key: 'CONTAINER_REGISTRY_HOST  ', value: 'gcr.io').save"
```

Comment:
General comment on the script, you spoke that it will be transformed to proper structure.
Functions, conditions, etc.

Reply:
It will, but not in this version